plugins/linux-capture/CMakeLists.txt

```diff
@@ -46,37 +46,41 @@ set(linux-capture_LIBRARIES
 
 option(ENABLE_PIPEWIRE "Enable PipeWire support" ON)
 if(ENABLE_PIPEWIRE)
-	find_package(PipeWire REQUIRED)
-	find_package(Gio REQUIRED)
+	find_package(PipeWire QUIET)
```

Comment:
Move the package checks outside the ENABLE_PIPEWIRE if, and then set the ENABLE_PIPEWIRE so you dont need to increase indention like this?

Reply:
And probably use two variables to ensure that the build actually fails if ENABLE_PIPEWIRE was set and pipewire was not found. As is it will just have a little status message and configure successfully if you asked for pipewire and it was not found.

Reply:
I've now simplified the code with these suggestions.

Reply:
Err I mean if we set ENABLE=ON and we cant find packages we should fail, not disable pipewire. If we set ENABLE=OFF we should not magically enable either. If we want to add another parameter for AUTO we can do that with this kind of check.